interface-elements-for-web/articles/menu.md

```diff
@@ -4,11 +4,12 @@ author: Svetlana Nikulina
 legacyId: 9382
 ---
 # Menu
-Menu is used to navigate you through items.
+The menu is used to navigate through items.
```

Comment:
"The menu is used to navigate through items." I'm not sure this is the best description of what the 'Menu' is.

Reply:
Modified.